meta-lmp-support/recipes-bsp/u-boot/u-boot-xlnx_%.bbappend

```diff
@@ -0,0 +1,5 @@
+COMPATIBLE_MACHINE = "uz"
+
+# do_deploy is skipped if valid in the shared state cache and copied to DEPLOY_DIR_IMAGE,
+# which is missed in install of openembedded-core/u-boot.inc
+addtask deploy before do_install
```

Comment:
Looks a bit strange, almost as if this should be asked via the xilinx mailing list...

Reply:
I.e. saying a bug on the Xilinx side.

Reply:
Actually, it looks like an openembedded bug (not Xilinx) as commented in the code. To find out a correct fix for openembedded might take some time so trying if this would be enough to fix random failures in our CI. Since CI failures are occurring quite randomly let's merge this and watch for any failures some days.